khmer/__init__.py

```diff
@@ -74,9 +72,8 @@
 
 from khmer._khmer import FILETYPES
 
-#from khmer._oxli.graphs import _Counttable
```

Comment:
I like the idea of removing the `_BlahType`s. Only question is: why were they added in the first place and then also used in various "user facing" places instead of always using `BlahType`? If no one remembers then let's get rid of them :)